tools/python_api/src_cpp/include/cached_import/py_cached_modules.h

```diff
@@ -24,6 +24,23 @@ class DecimalCachedItem : public PythonCachedItem {
     PythonCachedItem Decimal;
 };
 
+class ImportLibCachedItem : public PythonCachedItem {
+
+    class UtilCachedItem : public PythonCachedItem {
+
+    public:
+        explicit UtilCachedItem(PythonCachedItem* parent)
+            : PythonCachedItem("util", parent), find_spec("find_spec", this) {}
```

Comment:
We prefer to use bracket initilization pattern. 
PythonCachedItem{"util", parent}
find_spec{"find_spec", this}